The job **`mingw-check`** failed! Check out the build log: [(web)](https://github.com/rust-lang/rust/actions/runs/11898888314/job/33156346893) [(plain)](https://github.com/rust-lang/rust/commit/ba7c16dde7e18c7229ecdcb413e3ee67fd042edc/checks/33156346893/logs)

<details><summary><i>Click to see the possible cause of the failure (guessed by this bot)</i></summary>

```plain
 Documenting std v0.0.0 (/checkout/library/std)
error: unresolved link to `fs::remove_dir_all`
    --> std/src/fs.rs:2742:51
     |
2742 | /// of its contents recursively, consider using [`fs::remove_dir_all`]
     |                                                   ^^^^^^^^^^^^^^^^^^ no item named `fs` in scope
     = note: `-D rustdoc::broken-intra-doc-links` implied by `-D warnings`
     = help: to override `-D warnings` add `#[allow(rustdoc::broken_intra_doc_links)]`

error: could not document `std`
error: could not document `std`
warning: build failed, waiting for other jobs to finish...
Command has failed. Rerun with -v to see more details.
  local time: Mon Nov 18 19:04:00 UTC 2024
  network time: Mon, 18 Nov 2024 19:04:00 GMT
##[error]Process completed with exit code 1.
Post job cleanup.
```

</details>

--- a/library/std/src/fs.rs
+++ b/library/std/src/fs.rs
@@ -2738,6 +2738,10 @@ pub fn create_dir_all<P: AsRef<Path>>(path: P) -> io::Result<()> {
 
 /// Removes an empty directory.
 ///
+/// If you want to remove a directory that is not empty, as well as all
+/// of its contents recursively, consider using [`remove_dir_all`]
+/// instead.
+///
 /// # Platform-specific behavior
 ///
 /// This function currently corresponds to the `rmdir` function on Unix

